frontend/packages/kubevirt-plugin/src/plugin.tsx

```diff
@@ -62,6 +62,7 @@ const virtualMachineConfigurations = (namespace: string): FirehoseResource[] =>
       kind: models.VirtualMachineModel.kind,
       namespace,
       prop: 'virtualmachines',
+      optional: true,
```

Comment:
will it be better to set `virtualmachineinstances` , `virtualmachinetemplates`, `migrations` as well as `optional`?